front/src/common/api/generatedEditoastApi.ts

```diff
@@ -3577,12 +3577,17 @@ export type Conflict = {
   conflict_type: 'Spacing' | 'Routing';
   /** Datetime of the end of the conflict */
   end_time: string;
+  /** List of paced train occurence ids involved in the conflict */
+  paced_train_occurrence_ids?: {
+    index: number;
+    paced_train_id: number;
+  }[];
```

Comment:
If I understand correctly, this gives, for each paced train involved in a conflict, its id and the index of the occurrence responsible of the conflict ?
Maybe the doc above should reflect that instead of only mentioning "train occurence ids" (by the way, occurrence comes with 2 `r` in english 🙂).
I also think that the property name isn't accurate enough. Occurrences don't have ids but indexes.
```suggestion
  /** List of paced train occurrence ids involved in the conflict */
  paced_train_occurrence_ids?: {
    index: number;
    paced_train_id: number;
  }[];
```

Also what if multiple occurrences are responsible for the conflict ? (can this happen ?) Do we want to provide the first one with conflict or all of them ?

Reply:
I'm adding the info. Thanks :)
If multiple occurrences are responsible for the conflict, then all of them will be provided 
for exemple 

[{index: 1, paced_train_id: 4}, {index:4, paced_train_id:4}]